coveo-testing/tests_testing/test_test_id.py

```diff
@@ -11,6 +11,8 @@
 from coveo_testing.parametrize import parametrize
 from coveo_testing.temporary_resource.unique_id import TestId, unique_test_id
 
+_ = unique_test_id  # mark fixtures are used

```

Comment:
Did you mean

```suggestion
_ = unique_test_id  # mark fixtures as used

```
?
(I feel like an auto-correct :laughing:)